src/platform.ts

```diff
@@ -46,13 +50,22 @@ export class MideaPlatform implements DynamicPlatformPlugin {
 
     // Add default config values
     this.platformConfig = defaultsDeep(config, defaultConfig);
+    // Enforce min/max values
+    this.platformConfig.refreshInterval = Math.max(0, Math.min(this.platformConfig.refreshInterval ?? 30, 86400));
```

Comment:
It's not neccessary to handle if the refreshInterval/heartbeatInterval is undefined or not. Also, there is a limit in the config schema for both of these fields. If a user sets the value above the maximum by editing the raw config.json, then it's their responsibility in my opinion.

Reply:
I might be nitpicking here a bit :D but don't want to have unneccessary lines in the code.

Reply:
Yes, you're right. Will fix next time I commit.